website/src/pages/documents/document.page.tsx

```diff
@@ -331,12 +336,20 @@ export const DocumentTitleHeader = (p: {
       {p.doc.title}
       {p.doc.date && ` (${p.doc.date.year})`}{" "}
     </h1>
+        
+    
+
     <div className={css.bottomPadded}>
       {p.showDetails ? (
         <Link href={documentDetailsRoute(p.doc.slug!)}>View Details</Link>
       ) : (
         <Link href={documentRoute(p.doc.slug!)}>View Contents</Link>
       )}
+      {!p.doc.audioRecording && (
+        <div id="no-audio-message">
+        {!isMobile ? <a><b>No Audio Available</b></a> : null}
```

Comment:
This might be a nitpick, but please use the more semantically appropriate strong element here in place of the bring attention b element.